services/continuous_querier/config.go

```diff
@@ -24,13 +25,23 @@ type Config struct {
 	// every minute, this should be set to 1 minute. The default is set to '1s' so the interval
 	// is compatible with most aggregations.
 	RunInterval toml.Duration `toml:"run-interval"`
+
+	// CQ timeout for how long to wait for the previous query to complete, the default is `1h` so
+	// the timeout is compatible with most CQs that are run.
+	CQTimeout toml.Duration `toml:"query-timeout"`
+
+	// CQ denominator of how many (out of the total CQs configured) to run concurrently
+	// default: 2 (50%)
+	CQDenominator int `toml:"query-denominator"`
```

Comment:
@jsternberg Actually, would it make more sense to use a limit of half the available cores to try and ensure that this won't introduce a ton of resource contention for larger installations with many CQs configured?